components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/cache/EntitlementEngineCache.java

```diff
@@ -63,21 +64,20 @@ public static EntitlementEngineCache getInstance() {
 
     private Cache<Integer, EntitlementEngine> getEntitlementCache() {
         Cache<Integer, EntitlementEngine> cache;
-        CacheManager cacheManager = Caching.getCacheManagerFactory().
-                getCacheManager(ENTITLEMENT_ENGINE_CACHE_MANAGER);
+        CacheManager cacheManager = Caching.getCacheManagerFactory().getCacheManager(ENTITLEMENT_ENGINE_CACHE_MANAGER);
         if (cacheManager != null) {
-            if (cacheBuilder == null){
+            if (cacheBuilder == null) {
                 Properties properties = EntitlementServiceComponent.getEntitlementConfig().getEngineProperties();
                 String engineCachingInterval = properties.getProperty(PDPConstants.ENTITLEMENT_ENGINE_CACHING_INTERVAL);
-                long entitlementEngineCachingInterval = 900;
-                if (engineCachingInterval != null){
-                    try{
+                long entitlementEngineCachingInterval = DEFAULT_ENTITLEMENT_ENGINE_CACHING_INTERVAL;
+                if (engineCachingInterval != null) {
+                    try {
                         entitlementEngineCachingInterval = Long.parseLong(engineCachingInterval);
-                    }catch (NumberFormatException e){
+                    } catch (NumberFormatException e) {
```

Comment:
May be we can add a log (similar to the below debug log) here to mention this behaviour.